templates/default/config.php.erb

```diff
@@ -47,3 +47,6 @@ $config['rrdtool_version'] = '1.5.5';
 
 $config['geoloc']['latlng'] = true;
 $config['geoloc']['engine'] = "google"; //Only one available at present
+
+if(file_exists( __DIR__  . '/config.custom.php'))
```

Comment:
https://github.com/librenms/librenms/blob/master/doc/Developing/Code-Guidelines.md => "A space must be used both before and after the parenthesis and also surrounding the condition operator."

Reply:
Missing brackets